.misc/generate_dictionary.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# generate_dictionary.py
+#
+# Generates custom documentation dictionary for SpellCheckBear
```

Comment:
A module docstring should appear immediately after the hashbang, in double quotes.